cliconfig/config_test.go

```diff
@@ -75,14 +80,105 @@ func TestEmptyJson(t *testing.T) {
 	}
 
 	// Now save it and make sure it shows up in new form
-	err = config.Save()
+	saveConfigAndValidateNewFormat(t, config, tmpHome)
+}
+
+func TestOldInvalidsAuth(t *testing.T) {
+	invalids := map[string]string{
+		`username = test`: "The Auth config file is empty",
+		`username
+password
+email`: "Invalid Auth config file",
+		`username = test
+email`: "Invalid auth configuration file",
+		`username = am9lam9lOmhlbGxv
+email`: "Invalid Auth config file",
+	}
+
+	if runtime.GOOS == "windows" {
```

Comment:
The original test on old config are manipulating the ``HOME`` environment variable were doing that. I'm gonna try something real quick to not skip them on windows though :wink:.